Tools/autotest/arducopter.py

```diff
@@ -4648,12 +4648,11 @@ def fly_rangefinder_drivers(self):
             ("benewake_tf03", 27),
         ]
         while len(drivers):
-            do_drivers = drivers[0:3]
```

Comment:
The commit message on this doesn't make a great  deal of sense to me.

Reply:
we loose one port out of 3 available among the ones mentioned here: https://github.com/ArduPilot/ardupilot/pull/13746/files/cab45bd29320953ace29390ce05c8452c3bc839d#diff-45a8b9794e6771b33f8ac661964a893fL100